cinch/roles/jenkins_master/templates/setenvvars.groovy

```diff
@@ -0,0 +1,22 @@
+import jenkins.*;
```

Comment:
I couldn't really grok the way this one was working. It seemed very counter-intuitive to how I would think setting the variables would work, so I left it as-is.